node/narwhal/tests/bft_e2e.rs

```diff
@@ -123,3 +124,50 @@ async fn test_quorum_break() {
     // Check the nodes have stopped advancing through the rounds.
     assert!(network.is_halted().await);
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_leader_election_consistency() {
+    // The round at which to start checking for leader consistency.
```

Comment:
is this the minimum possible round to start the checks? if so, it should be commented on

Reply:
I tried starting on 0 and got some inconsistency issues, probably starting from round 2 might already be fine. I just added a tiny bit of buffer.